packages/crayons-core/src/components/checkbox/checkbox.stories.mdx

```diff
@@ -42,28 +42,28 @@ import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
 ### Checkbox with Text
```

Comment:
The story headings are very confusing. We seem to be using the word _Text_ to refer to what is actually the `label`. Further down, we have a variant called _Checkbox with Text  and Label_ in which the _Text_ seems to refer to the `label` prop and the _Label_ seems to refer to the `description` slot. Let's fix this.

Reply:
Apologies for the incorrect convention, I modified the component within stories but forgot to modify the headings. Have accommodated in latest commit. Thank you